spec/services/catalog/copy_portfolio_spec.rb

```diff
@@ -23,6 +23,7 @@
 
         expect(new.icon_id).not_to eq portfolio.icon_id
         expect(new.icon.image_id).to eq portfolio.icon.image_id
+        expect(new.icon.restore_to).to eq new_portfolio
```

Comment:
Do we have a similar test for portfolio items?

Reply:
Yep - my bad. Updated.